UnityProject/Assets/Scripts/Items/Weapons/PowerSink.cs

```diff
@@ -156,5 +161,14 @@ public override void ReceiveSignal(SignalStrength strength, ISignalMessage messa
 		{
 			ToggleActivity();
 		}
+
+		private IEnumerator BeepBeep()
+		{
+			while (isCharging && gameObject != null)
+			{
+				SoundManager.PlayNetworkedAtPos(beepSound, gameObject.AssumedWorldPosServer());
+				yield return WaitFor.Seconds(2f);
```

Comment:
would be nice if it slowly sped up the closer to exploding it was

Reply:
I like that idea, gonna add it once im done with another PR

Reply:
ok cool